### ISSUE TITLE
Вернуть жидкие столбцы

### ISSUE THREAD

**Issue by tigran123** [tigran123/urantia-book-explorer#33]
Хотя бы для результатов поиска, но желательно для всех столбцов.

**Comment by alyuev** [comment]
Жидкие - это имеется в виду - имеющие возможность регулировать ширину столбца вручную?

**Comment by tigran123** [comment]
Да, на языке jQuery это метод `.resizable()` применимый почти к любому элементу. НО.... наивно его "просто включить" нельзя, ибо очень многие вещи от этого сломаются. Т.е. надо очень осторожно исследовать как это лучше сделать.
